dbt/include/snowflake/macros/catalog.sql

```diff
@@ -42,7 +42,7 @@
         table_schema as "table_schema",
         table_name as "table_name",
         case
-            when is_dynamic = 'YES' and table_type = 'BASE TABLE' THEN 'DYNAMIC TABLE'
+            when (is_dynamic = 'YES' OR is_dynamic = 'Y') and table_type = 'BASE TABLE' THEN 'DYNAMIC TABLE'
```

Comment:
more canonical imho
```suggestion
            when is_dynamic IN ('YES', 'Y') and table_type = 'BASE TABLE' THEN 'DYNAMIC TABLE'
```

Reply:
Can be actually be less efficient for large sets of things, but otherwise yep, I'd it's the nicer syntax and perfectly fine in this application.

Reply:
will update

Reply:
updated both query places to use this method